src/main/lib/access-control-transformer.js

```diff
@@ -0,0 +1,64 @@
+const lodash = require('lodash');
```

Comment:
Do you need to add lodash as a dependancy in the package.json?

Reply:
thats good spot, i have unconsciously piggyback on transitive dependency